packages/npm-dependencies-validation/src/index.ts

```diff
@@ -0,0 +1,68 @@
+import { spawnCommand } from "./utils/npmUtil";
```

Comment:
can we rename this file to something more meaningful?
perhaps named after the single function it exports?

Reply:
imports should be ordered according to these rules
- https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

Reply:
Done